memcached-exporter.yaml

```diff
@@ -29,6 +29,15 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: memcached-exporter-bitnami-compat
+    description: "compat package with bitnami/memcached-exporter image"
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: memcached-exporter
+          version-path: 0/debian-11
```

Comment:
There is nothing in the run section is that expected

Reply:
I didn't add the run section because there is nothing in the /opt/bitnami folder in memcached-exporter here:

- https://github.com/bitnami/containers/tree/bf0ed9f4c22f3390ed36a20bee8565190c63dbd2/bitnami/memcached-exporter/0/debian-11/prebuildfs/opt/bitnami

Reply:
I don't think that we need to do anything in the run section, creating symlinks, files/folders etc.